cmd/registry/cmd/compute/vocabulary/vocabulary-proto.go

```diff
@@ -41,7 +41,7 @@ func NewVocabularyFromZippedProtos(b []byte) (*metrics.Vocabulary, error) {
 	}
 
 	for _, f := range r.File {
-		if strings.HasSuffix(f.Name, ".proto") {
+		if !strings.HasSuffix(f.Name, ".proto") {
```

Comment:
@theganyo This is the error -- introduced in a big refactor a while ago, it was preventing the proto vocabulary computation from finding anything.